website/docs/plugins/nevergrad_sweeper.md

```diff
@@ -59,12 +59,14 @@ batch_size=4
 
 To run hyperparameter search and look for the best parameters for this function, clone the code and run the following command in the `plugins/hydra_nevergrad_sweeper` directory:
 ```bash
-python example/dummy_training.py -m
+python example/my_app.py.py -m
 ```
 
 You can also override the search space parametrization:
 ```bash
-python example/dummy_training.py -m db=mnist,cifar batch_size=4,8,16 lr=log:0.001:1 dropout=0:1
+python example/my_app.py.py -m \
+db=mnist,cifar batch_size=4,8,16 \
+'lr=tag(log, interval(0.001, 1))' 'dropout=interval(0,1)'
```

Comment:
formatting (and full --multirun instead of -m to make it clear).
```suggestion
python example/my_app.py.py --multirun db=mnist,cifar batch_size=4,8,16 \
'lr=tag(log, interval(0.001, 1))' 'dropout=interval(0,1)'
```
Also, can you add a word about quoting the parameters?